### PR TITLE
Tsc project roles update

### PR DESCRIPTION
**Pull request by ioannisg**
This pull request contains the recommendations from the Process Improvement Forum for minor amendments to the Zephyr TSC Project Roles section. 

To be merged, the PR requires approval by the TSC.

### CODE REVIEW

File: doc/contribute/project_roles.rst
Comment (galak) [inline doc/contribute/project_roles.rst]: Remove this bullet and just explicit list the roles in the second bullet and remove the 'leadership role' term.

File: doc/contribute/project_roles.rst
Comment (galak) [inline doc/contribute/project_roles.rst]: replace Contributors with individual.

File: doc/contribute/project_roles.rst
Comment (galak) [inline doc/contribute/project_roles.rst]: Make it clear that the process is a TSC member would raise a motion for removal and that would be voted.

File: doc/contribute/project_roles.rst
Comment (ioannisg) [inline doc/contribute/project_roles.rst]: "ensure" instead of "maintaining"

File: doc/contribute/project_roles.rst
Comment (ioannisg) [inline doc/contribute/project_roles.rst]: @kestewart could you check this amendment - whether it is fine for you?

File: doc/contribute/project_roles.rst
Comment (ioannisg) [reply doc/contribute/project_roles.rst]: Agreed, updating this

File: doc/contribute/project_roles.rst
Comment (ioannisg) [reply doc/contribute/project_roles.rst]: agreed, updating this as well

File: doc/contribute/project_roles.rst
Comment (ioannisg) [reply doc/contribute/project_roles.rst]: Agreed

**Review by ioannisg — COMMENTED** [review]
@nashif @galak @MaureenHelm @jhedberg @carlescufi the PR is updated based on the discussion feedback in this week's Process meeting
https://docs.google.com/document/d/1VPDfXGEK_sOOIQYHK-95ROZUt0_BHgb7U-YiGz1YUqk/edit#heading=h.lmeah8mweht8

**Comment by ioannisg** [comment]
Keeping this as draft PR, until the TSC has approved the changes.

File: doc/contribute/project_roles.rst
Comment (nashif) [inline doc/contribute/project_roles.rst]: > Release Engineering team members are expected to be available,
>  from time to time, for serving as Release Managers.

How about:
* Members of the release engineering team are expected to fill the release engineering manager role based on a defined cadence and selection process.

**Review by nashif — CHANGES_REQUESTED** [review]

File: doc/contribute/project_roles.rst
Comment (ioannisg) [reply doc/contribute/project_roles.rst]: What exactly? Can you help?

File: doc/contribute/project_roles.rst
Comment (ioannisg) [reply doc/contribute/project_roles.rst]: This was based on your feedback @nashif

File: doc/contribute/project_roles.rst
Comment (ioannisg) [reply doc/contribute/project_roles.rst]: Yes, sure; it is more accurate, actually, based on how we discussed it.

Comment (ioannisg) [reply doc/contribute/project_roles.rst]: Updated this now.

File: doc/contribute/project_roles.rst
Comment (nashif) [reply doc/contribute/project_roles.rst]: there is this second bullet that has the wrong indentation, the first bullet does not say anything....

We do need some leading test, jumping into bullets like this does not give context and it is difficult to understand what is this section about.

File: doc/contribute/project_roles.rst
Comment (nashif) [reply doc/contribute/project_roles.rst]: it is already covered in the second bullet, no need to duplicate this.

**Review by nashif — CHANGES_REQUESTED** [review]

File: doc/contribute/project_roles.rst
Comment (ioannisg) [reply doc/contribute/project_roles.rst]: I don't think there is something wrong with the indentation - this list of roles are an enumeration

File: doc/contribute/project_roles.rst
Comment (ioannisg) [reply doc/contribute/project_roles.rst]: Then, I can remove. :)

File: doc/contribute/project_roles.rst
Comment (nashif) [reply doc/contribute/project_roles.rst]: image

The second level bullet is misplaced, you probably should do the listing of roles differently

Comment (ioannisg) [reply doc/contribute/project_roles.rst]: I can't see how this is a problem: seems to be like this in the entire Zephyr Documentation

**Comment by ioannisg** [comment]
This is now updated based on the feedback received in Week-13 Process Forum meeting.

File: doc/contribute/project_roles.rst
Comment (ioannisg) [reply doc/contribute/project_roles.rst]: Fixed now @nashif

File: doc/contribute/project_roles.rst
Comment (kestewart) [reply doc/contribute/project_roles.rst]: We'd need to update the charter to be explicit here.    Can you help me understand the background on why the existing TSC voting members as a statement isn't felt to be sufficient?

**Comment by nashif** [comment]
are you expecting us to review this or is this just a draft?

File: doc/contribute/project_roles.rst
Comment (ioannisg) [reply doc/contribute/project_roles.rst]: The rationale behind this minor change is the following: If TSC should be allowed to add additional Maintainers to the committee, this needs to be done in a way that it is compatible with the Charter, which currently states that "community members may become TSC voting members at a rate no more than 1 per quarter".

This rule applies now, but may be changed (or even removed) in the future. Therefore, what I wanted here is to state - in a generic way- that TSC should be allowed to add new members in the committee as long as this is done in the way the charter specifies it.

**Comment by ioannisg** [comment]
> are you expecting us to review this or is this just a draft?

I guess, no need to be reviewed by the people that have given already their feedback in the Process Forum meetings. Others are welcome to file comments. This is draft because it has not yet be approved by TSC.

**Review by nashif — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**
